core/src/com/unciv/models/ruleset/unique/UniqueParameterType.kt

```diff
@@ -18,6 +18,10 @@ enum class UniqueParameterType(val parameterName:String) {
         private val knownValues = setOf("Wounded", "Barbarians", "City-State", "Embarked", "Non-City")
         override fun getErrorSeverity(parameterText: String, ruleset: Ruleset):
                 UniqueType.UniqueComplianceErrorSeverity? {
+            if ('{' in parameterText) // "{filter} {filter}" for and logic
+                return parameterText.removePrefix("{").removeSuffix("}").split("} {")
```

Comment:
Does this still work for languages that don't use spaces?

Reply:
And I thought that code I copied there was yours... But no, blame says Yair's.
This is just a counterpart to MapUnit.matchesFilter so the unique-auto-checks will accept the same input as the filter itself.
And this is pre-translation, so... Should not be a step back? But that's why I value input - sometimes I get the feeling to have missed some aspect and can't put my finger on it, this one especially so.

Reply:
This looks good to me!
The translation of said parameters should be dealt with, but definitely outside this PR